conduit/conduit.cabal

```diff
@@ -59,7 +59,7 @@ test-suite test
                    , base
                    , hspec >= 1.3
                    , QuickCheck >= 2.7
-                   , transformers
+                   , transformers >= 0.4
```

Comment:
Thanks for the info. I wasn't aware of the existence of transformers-compat. Would you prefer I add another commit, or squash the changes into this one?